### PR TITLE
feat: run all tests in parallel

### PR DESCRIPTION
**Pull request by xudong963**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

1. use `sandbox_tenant` to isolate each test file.
2. run all test files in parallel.

I did a simple test, running crdb suit. **Run time will be reduced twice**

Closes #issue

### CODE REVIEW

File: tests/sqllogictests/src/client/clickhouse_client.rs [hunk 1]
```diff
@@ -43,6 +43,7 @@ impl ClickhouseHttpClient {
     }
 
     pub async fn query(&mut self, sql: &str) -> Result<DBOutput> {
+        println!("Running sql with clickhouse client: [{}]", sql);
```
Comment (Xuanwo) [inline tests/sqllogictests/src/client/clickhouse_client.rs:46]: Please use `debug` or `info`.

Comment (xudong963) [reply tests/sqllogictests/src/client/clickhouse_client.rs:46]: If we use them, we'll get some noise from sqllogictest-rs.

I think those logs are basic and useful, maybe it's better to print them always.

File: tests/sqllogictests/src/main.rs [hunk 2]
```diff
@@ -142,7 +142,10 @@ async fn create_databend(client_type: &ClientType) -> Result<Databend> {
             client = Client::Clickhouse(ClickhouseHttpClient::create()?);
         }
     }
-    client.create_sandbox().await?;
+    let suites = SqlLogicTestArgs::parse().suites;
+    if suites != "tests/fuse-compat/compat-logictest" {
```
Comment (xudong963) [inline tests/sqllogictests/src/main.rs:146]: To take care of compatibility testing, `set sandbox_tenant` was not supported in past versions.

Maybe we should find a better way to test compat.  Such as use the traditional test way not using sqllogictest.

cc @Xuanwo @sundy-li

**Comment by xudong963** [comment]
@mergify update

File: tests/sqllogictests/src/main.rs (hunk 2 above)
Comment (sundy-li) [reply tests/sqllogictests/src/main.rs:146]: ok

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**